dspace/config/crosswalks/DIM2DataCite.xsl

```diff
@@ -50,12 +53,20 @@
             <!-- 
                 DataCite (1)
                 Template Call for DOI identifier.
-            --> 
-            <xsl:apply-templates select="//dspace:field[@mdschema='dc' and @element='identifier' and starts-with(., 'http://dx.doi.org/')]" />
+                Occ: 1
+            -->
+            <!--
+                dc.identifier.uri may contain more than one DOI, e.g. if the
+                repository contains an item that is published by a publishing 
```

Comment:
I'm not that familiar with the current implementation in DSpace. What I'm saying is that if an item has a DOI, regardless of field DSpace uses to store it, it should not be possible to mint another DOI (to shoot yourself in the foot as a publisher).